<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneChangerConfig</class>
 <widget class="QWidget" name="SceneChangerConfig">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vertical_layout">
   <item>
    <layout class="QHBoxLayout" name="command_input_layout">
     <item>
      <widget class="QLabel" name="command_label">
       <property name="text">
        <string>Command</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="command_line_edit">
       <property name="text">
        <string>scene</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="enabled_users_input_layout">
     <item>
      <widget class="QLabel" name="who_can_use_label">
       <property name="text">
        <string>Who can use it?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mod_check_box">
       <property name="text">
        <string>Mods</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="vip_check_box">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>VIP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="sub_check_box">
       <property name="text">
        <string>Subs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chatter_check_box">
       <property name="text">
        <string>Chatter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="scene_change_input_layout">
     <item>
      <widget class="QLabel" name="from_label">
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="from_combo_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="to_label">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="to_combo_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="scene_changes_widget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
